docs/packages.rst

```diff
@@ -117,42 +125,59 @@ community.
 General Best Practices for Open Source
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. Use an open-source license. See `Jake VanderPlas' article on
-   licensing scientific
-   code <http://www.astrobetter.com/blog/2014/03/10/the-whys-and-hows-of-licensing-scientific-code/>`__
-   or `these more general guidelines <https://choosealicense.com/>`__
-2. Use version control for source code (for example, on
-   `github <http://github.org>`__)
-3. Provide thorough test coverage and continuous integration of testing
-4. Maintain comprehensive documentation
-5. Establish a `code of
-   conduct <https://opensource.guide/code-of-conduct/>`__ for
-   contributors
-
-The `open-source guide <https://opensource.guide/>`__ provides some
-great advice on building and maintaining open-source projects.
+How to develop and maintain an open-source project is a large topic that extends beyond pangeo.
+The `pyOpenSci Package Development Guide`_ provides a comprehensive guide.
```

Comment:
@TomAugspurger this is great. our package guide right now if you go to that link is really sparse. right now i have a [HUGE documentation pr](https://github.com/pyOpenSci/python-package-guide/pulls) that will be merged next week. And then after that i have a packaging / build tools oprn PR that will be merged but after another round of community review. So i'm guessing our guide won't be stable for another 2-3 month as each PR takes some time to review etc... 

do you want to potentially add a note about that or how would you like to handle that?